client/components/category-pill-navigation/index.tsx

```diff
@@ -11,6 +11,7 @@ type CategoryPillNavigationProps = {
 		icon: React.ReactElement< typeof Icon >;
 		label: string;
 		link: string;
+		isActive?: boolean;
```

Comment:
I commented yesterday https://github.com/Automattic/wp-calypso/pull/88343#issuecomment-2003243768 that I think we should reconsider the name of the `selectedCategory`. With this change, I think that's even more relevant, because theoretically, this component can have one active `button` _and_ one active `link` now.

One option could be to swap `selectedCategory` for something like `activeHref` and compare `button.link` and `link.link` with `activeHref`.

I'm fine with saving this for later, though! In fact, I created https://github.com/Automattic/dotcom-forge/issues/6159 to track that work and unblock us from merging this.